ci/build.sh

```diff
@@ -4,4 +4,12 @@ set -e -u -x -o pipefail
 
 cd "$(dirname "$0")/.."
 
-spatial build --log_level debug
+PLATFORM=""
+if [[ "$(uname -s)" == "Linux" ]]; then
+  PLATFORM="linux"
+elif [[ "$(uname -s)" == "Darwin" ]]; then
+  PLATFORM="macos"
+else
+  PLATFORM="windows"
+fi
+spatial build --target $PLATFORM --log_level=debug
```

Comment:
Inconsistent arguments. Log level tends to be without `=` in other example projects.

Reply:
Changing other projects to match this style and the documentation